webrender/src/renderer.rs

```diff
@@ -1296,6 +1374,20 @@ impl Renderer {
                                       options.precache_shaders)
         };
 
+        let brush_image_a8 = try!{
+            BrushShader::new("brush_image",
+                             &mut device,
+                             &["ALPHA"],
```

Comment:
hmm, the `alpha` shader already gets this feature inside the `new()`. Is this a collision of features names, or just to reflect the fact that the opaque version of the shader makes no sense here?

Reply:
Well spotted - I made the feature names a bit more specific.